src/mtype.d

```diff
@@ -3492,7 +3492,7 @@ public:
             case Tcomplex80:
             case Timaginary80:
             case Tfloat80:
-                fvalue = Port.ldbl_max;
+                fvalue = real.max;
```

Comment:
@yebblies I just replaced a reference to `Port.ldbl_max`, which was an *unconditional* `enum` to `real.max;`, with a direct reference to its definition of `real.max;`. Are you saying that I introduced a bug or that that code was already incorrect?